README.md

```diff
@@ -16,7 +16,7 @@ There are several examples included in this repository for various SDKs. If you
 Ensure you have Node 16.x installed.
 
 1. Clone this repo
-2. Run `yarn` from the root directory. SDKs and examples should now be set up to run via Nx.
+2. Run `pnpm` from the root directory. SDKs and examples should now be set up to run via Nx.
```

Comment:
Do we need to add install instructions on how to setup `pnpm`? is it in the brew-install script?